api/views/admin_postings.py

```diff
@@ -86,3 +86,23 @@ def get_specific_posting(posting_id):
 
     except Exception as e:
         return make_response(return_exception(e), 400)
+
```

Comment:
Can you get rid of all the code from line 89 to 108? I believe Claire is already doing this!